### PR TITLE
fix: Fix building "rhc" using make

### PR DESCRIPTION
**Pull request by jirihnidek**
* It is possible to build rhc using `make`
* It is also possible to clean repository using `make clean`, when some generated file has been already deleted.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -148,8 +147,8 @@ dist:
 .PHONY: clean
 clean:
 	go mod tidy
-	rm $(BIN)
-	rm $(DATA)
+	rm -f $(BIN)
```
Comment (subpop) [inline Makefile:150]: Maybe adding the `-` prefix to these lines so they don't cause `make` to exit if the command exits with a non-zero exit code?

Comment (jirihnidek) [reply Makefile:150]: Interesting: https://www.gnu.org/software/make/manual/make.html#index-_002d-_0028in-recipes_0029, but it still prints errors about non-existing files to `stdout`, and I think we want to suppress printing errors too.

Comment (subpop) [reply Makefile:150]: I think you'd have to redirect stderr to /dev/null in that case.

Comment (jirihnidek) [reply Makefile:150]: I know, but I consider current solution as more straightforward.

Comment (jirihnidek) [reply Makefile:150]: @subpop Hi, are you requesting your proposal or is my solution good enough?

Comment (subpop) [reply Makefile:150]: Oh. This is fine with me. I'm not concerned with suppressing output.

**Review by subpop — APPROVED** [review]

**Pull request merged**